config/webpack/plugins/htmlRenderPlugin.js

```diff
@@ -13,11 +13,7 @@ const {
   defaultClientEntry
 } = require('../../../context');
 
-const createPublicUrl = (publicPath, asset) => {
-  const host = publicPath.endsWith('/') ? publicPath : `${publicPath}/`;
-
-  return `${host}${asset}`;
-};
+const createPublicUrl = (publicPath, asset) => `${publicPath}${asset}`;
```

Comment:
Is this function even useful anymore?

Reply:
We concatenate in multiple places so figured it's worth keeping. Happy to inline if you prefer?